versions.yaml

```diff
@@ -122,7 +122,7 @@ assets:
     architecture:
       aarch64:
         name: &default-image-name "ubuntu"
-        version: &default-image-version "latest"
+        version: &default-image-version "jammy"
```

Comment:
Note that [latest LTS](https://fridge.ubuntu.com/2024/04/12/ubuntu-24-04-lts-noble-numbat-beta-released/) is expected later this week. It is worth to hold on a bit more and bump to `"noble"` when it is GA.